src/main/java/com/eternalcode/check/CheckNotificationTask.java

```diff
@@ -1,53 +1,45 @@
 package com.eternalcode.check;
 
 import com.eternalcode.check.config.implementation.MessagesConfig;
-import com.eternalcode.check.config.implementation.PluginConfig;
+import com.eternalcode.check.notification.Notification;
+import com.eternalcode.check.notification.NotificationAnnoucer;
 import com.eternalcode.check.user.CheckedUser;
 import com.eternalcode.check.user.CheckedUserService;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
 import panda.utilities.text.Formatter;
 
-import java.time.Duration;
-import java.util.UUID;
+import java.util.ArrayList;
 
 public final class CheckNotificationTask implements Runnable {
 
-    private final MessagesConfig messages;
-    private final PluginConfig config;
     private final CheckedUserService checkedUserService;
-    private final NotificationAnnouncer announcer;
-
-    private final Duration stay, fadeOut, fadeIn;
+    private final NotificationAnnoucer announcer;
+    private final MessagesConfig messages;
+    private final Server server;
 
-    public CheckNotificationTask(MessagesConfig messages, PluginConfig config, CheckedUserService checkedUserService, NotificationAnnouncer announcer) {
-        this.messages = messages;
-        this.config = config;
+    public CheckNotificationTask(CheckedUserService checkedUserService, NotificationAnnoucer announcer, MessagesConfig messages, Server server) {
         this.checkedUserService = checkedUserService;
         this.announcer = announcer;
-
-        this.stay = this.config.settings.title.stay;
-        this.fadeOut = this.config.settings.title.fadeOut;
-        this.fadeIn = this.config.settings.title.fadeIn;
+        this.messages = messages;
+        this.server = server;
     }
 
     @Override
     public void run() {
-        for (CheckedUser user : this.checkedUserService.getUsers()) {
-            UUID userUniqueId = user.getUniqueId();
+        for (CheckedUser user : new ArrayList<>(this.checkedUserService.checkedUsers())) {
+            Player player = this.server.getPlayer(user.getUniqueId());
+
+            if (player == null) {
+                continue;
```

Comment:
Dlaczego `continue;`, a nie np. `return;`?

Reply:
Tak działa pętla for each?